notebooks/itslive/itslive.ipynb

```diff
@@ -0,0 +1,234 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div align=\"center\"> \n",
+    "<img src='./img/header.png'/>\n",
+    "</div>\n",
+    "\n",
+    "## [Global Ice Velocities](https://its-live.jpl.nasa.gov/)\n",
+    "    \n",
+    "The Inter-mission Time Series of Land Ice Velocity and Elevation (ITS_LIVE) project facilitates ice sheet, ice shelf and glacier research by providing a globally comprehensive and temporally dense multi-sensor record of land ice velocity and elevation with low latency.\n",
+    "\n",
+    "Scene-pair velocities generated from satellite optical and radar imagery.\n",
+    "\n",
+    "* Coverage: All land ice\n",
+    "* Date range: 1985-present\n",
+    "* Resolution: 240m\n",
+    "* Scene-pair separation: 6 to 546 days\n",
+    "\n",
+    "\n",
+    "---\n",
+    "\n",
+    "* If you want to query our API directly using  your own software here is the OpenApi endpoint https://staging.nsidc.org/apps/itslive-search/docs\n",
+    "* For questions about this notebook and the dataset please contact users services at uso@nsidc.org\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#1: Now let's render our UI and pick up an hemisphere, if you update the hemisphere you need to execute the cell again.\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')\n",
+    "from itslive import itslive_ui\n",
+    "ui = itslive_ui('global')\n",
+    "ui.render()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#2: We build the parameters to query the ITS_LIVE Search API, we get the time coverage for our selected area\n",
+    "params = ui.build_params()\n",
+    "print(f'current parameters: {params}')\n",
```

Comment:
I get this error:

`
You need to select an area using the box tool or the polygon drawing tool
current parameters: None`

But there are no bbox or polygon options in the map - just "circle marker" option.